nope

rbpi:
configure: error: Could not link with SSL libs at /home/jenkins/workspace/LINUX-RBPI/tools/depends/xbmc-depends/arm-linux-gnueabihf

linux:
configure: error: Could not link with SSL libs at /home/jenkins/workspace/LINUX-64/tools/depends/xbmc-depends/x86_64-linux-gnu

I have rebased your pr to master and hopefully added a fix for this here:

https://github.com/Memphiz/xbmc/commits/add_ssl_to_mysql_for_mac_os_x_builds

Kicked a jenkins testbuild here:

http://jenkins.kodi.tv/view/Helpers/job/BuildMulti-All/1196/

--- a/tools/depends/target/Makefile
+++ b/tools/depends/target/Makefile
@@ -89,7 +89,7 @@ libcdio: $(ICONV)
 libplist: libxml2 $(ZLIB)
 libbluray: $(ICONV) libxml2
 libssh: libgcrypt openssl $(ZLIB)
-mysql: openssl
+mysql: openssl $(ZLIB)
 libzip: $(ZLIB)
 libpng: $(ZLIB)
 openssl: $(ZLIB)

--- a/tools/depends/target/mysql/06-fixsslcheck.patch
+++ b/tools/depends/target/mysql/06-fixsslcheck.patch
@@ -1,0 +1,11 @@
+--- config/ac-macros/ssl.m4.orig	2016-02-14 13:47:23.792296242 +0000
++++ config/ac-macros/ssl.m4	2016-02-14 13:47:05.464296846 +0000
+@@ -111,7 +111,7 @@
+   #
+   # Try to link with openSSL libs in <location>
+   #
+-  openssl_libs="-L$location/lib/ -lssl -lcrypto"
++  openssl_libs="-L$location/lib/ -lssl -lcrypto -lz -ldl"
+   MYSQL_CHECK_SSL_DIR([$openssl_includes], [$openssl_libs])
+ 
+   if test "$mysql_ssl_found" == "no"

--- a/tools/depends/target/mysql/Makefile
+++ b/tools/depends/target/mysql/Makefile
@@ -33,6 +33,7 @@ $(PLATFORM): $(TARBALLS_LOCATION)/$(ARCHIVE) $(DEPS)
 	cd $(PLATFORM); patch -Np1 -i ../03-mysqlclient-android.patch
 	cd $(PLATFORM); patch -p0 < ../04-strnlen.patch
 	cd $(PLATFORM); patch -p1 < ../05-mysqlclient-ios64.patch
+	cd $(PLATFORM); patch -p0 < ../06-fixsslcheck.patch
 	cd $(PLATFORM); autoconf
 	cd $(PLATFORM); $(CONFIGURE) 
 

--- a/tools/depends/target/mysql/Makefile
+++ b/tools/depends/target/mysql/Makefile
@@ -13,7 +13,8 @@ CONFIGURE=cp -f $(CONFIG_SUB) $(CONFIG_GUESS) .; \
 	--enable-thread-safe-client --with-extra-charsets=complex \
 	--with-named-thread-libs=-lc --with-named-curses-libs=-lncurses \
 	--with-libedit \
-	--without-server --without-bench --without-docs --without-man --disable-shared
+	--without-server --without-bench --without-docs --without-man --disable-shared \
+	--with-ssl=$(PREFIX) LIBS="-ldl -lz"
 
 LIBDYLIB=$(PLATFORM)/lib$(LIBNAME)/.libs/lib$(LIBNAME)client.a
 

